package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "abase2",
```

Comment:
If you want to test requiring abase into other projects, you should use `npm link` instead of publishing a dummy module to npm